config/routes.rb

```diff
@@ -17,4 +17,6 @@
   resources :projects do
     resources :tasks
   end
+
+  resources :project_information, path: "/projects/information"
```

Comment:
Should this be a child of the projects resource, so we get `projects/{id}/information"`?